includes/admin/class-wp-job-manager-admin-notices.php

```diff
@@ -398,22 +392,23 @@ public static function display_core_setup() {
 	}
 
 	/**
-	 * Displays the notice that informs about WPJM addon updates available.
+	 * Adds notice that informs about WPJM addon updates available if any.
 	 *
-	 * Note: For internal use only. Do not call manually.
+	 * @param array $notices Existing notices.
+	 *
+	 * @internal
+	 *
+	 * @return mixed
 	 */
-	public static function display_addon_update_available() {
-		if ( ! self::is_admin_on_standard_job_manager_screen( [ 'dashboard' ] ) ) {
-			return;
-		}
-
+	public static function maybe_add_addon_update_available_notice( $notices ) {
 		$updates = get_transient( 'wpjm_addon_updates_available', [] );
-
 		if ( ! empty( $updates ) ) {
-			$notice    = self::generate_notice_from_updates( $updates );
-			$notice_id = self::NOTICE_ADDON_UPDATE_AVAILABLE;
-			self::render_notice( $notice_id, $notice );
+			$notice = self::generate_notice_from_updates( $updates );
+			if ( ! is_null( $notice ) ) {
+				$notices[ self::NOTICE_ADDON_UPDATE_AVAILABLE ] = $notice;
```

Comment:
As is, if they ever dismiss the notice, it will never be shown again. Do we want this to be the behavior or should we generate the notice ID using the plugin updates available?